server/build.gradle

```diff
@@ -82,6 +82,9 @@ dependencies {
 
     Classpaths.inheritGrpcPlatform(project, 'api')
 
+    api('io.grpc:grpc-inprocess') {
```

Comment:
I proposed a change to downgrade this to testImplementation, by moving the one test class that needs it to src/main/test